huaweicloud/resource_huaweicloud_lb_member.go

```diff
@@ -190,12 +200,12 @@ func resourceMemberV2Update(d *schema.ResourceData, meta interface{}) error {
 	timeout := d.Timeout(schema.TimeoutUpdate)
 	err = waitForLBV2viaPool(lbClient, poolID, "ACTIVE", timeout)
 	if err != nil {
-		return err
+		return diag.FromErr(err)
 	}
 
 	logp.Printf("[DEBUG] Updating member %s with options: %#v", d.Id(), updateOpts)
 	//lintignore:R006
-	err = resource.Retry(timeout, func() *resource.RetryError {
+	err = resource.RetryContext(ctx, timeout, func() *resource.RetryError {
```

Comment:
does it need to retry after API changes?

Reply:
@chengxiangdong just keep it as is, will analyze it later.

Reply:
ok